mppm/player-tags/player-tags-create.md

```diff
@@ -4,11 +4,11 @@ title: Create a Player Tag
 description: Use these steps to create Player Tags to configure the player actions in Multiplayer Play Mode.
 ---
 
-To create a **Player Tag**, perform mthe following actions: 
+To create a **Player Tag**, perform the following actions: 
 
-1. In the **Unity Editor**, open the **MPPM** window by navigating to **Window** > **Multiplayer Play Mode**.
-2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create your new tag.
+1. In the **Unity Editor**, open the **Multiplayer Play Mode** window by navigating to **Window** > **Multiplayer Play Mode**.
+2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create a new tag.
```

Comment:
```suggestion
1. Select **Add Tags for Player**.
1. In the dropdown list, select **+ Create Tag**
```

Reply:
Does Docusaurus accept the "1. 1. 1." numbering like Github-flavored Markdown, or do we need to update all of the numbers here?

Reply:
In fact, we may want to replace this with the wording from https://docs-multiplayer.unity3d.com/mppm/current/player-tags/player-tags-attach/ for consistency.

Reply:
Updated list and instructions for consistency, thanks :)